fill/proto/interactive_filler_test.go

```diff
@@ -1,16 +1,87 @@
-package proto_test
+package proto
 
 import (
 	"testing"
 
+	"github.com/golang/protobuf/protoc-gen-go/descriptor"
 	"github.com/ktr0731/evans/fill"
-	"github.com/ktr0731/evans/fill/proto"
+	"github.com/ktr0731/evans/prompt"
 )
 
+type testPrompt struct {
+	input string
+}
```

Comment:
Can you embed `prompt.Prompt` and remove methods other than `Input`?

```suggestion
type testPrompt struct {
	prompt.Prompt
	
	input string
}
```

Reply:
great suggestion thank you!